README.md

```diff
@@ -158,15 +158,12 @@ A user can grant access to support agents by selecting the device from the
 Dashboard, and then selecting the 'Actions' tab in the left-hand sidepanel.
 Scrolling down the Actions page will show a list of actions, with the 'Grant
```

Comment:
```suggestion
Scrolling down the Settings page will show a list of actions, with the 'Grant
```

this whole part is unnecessarily wordy - e.g. do we really need to say 'scrolling the page will show a list', but perhaps a change for another time!